espnet/bin/lm_train.py

```diff
@@ -85,6 +85,8 @@ def get_parser():
                         help='dropout probability')
     parser.add_argument('--maxlen', type=int, default=40,
                         help='Batch size is reduced if the input sequence > ML')
+    parser.add_argument('--dump-dataset', type=str, default=None,
```

Comment:
I think it is better to rename or change to bool type to make it clear.

Reply:
Maybe `--dump-hdf5-path` is better?

Reply:
Sounds good :)